application/comit_node/src/http_api/rfc003/routes.rs

```diff
@@ -0,0 +1,345 @@
+use crate::{
+    http_api::{
+        self,
+        asset::HttpAsset,
+        ledger::HttpLedger,
+        problem::HttpApiProblemStdError,
+        rfc003::{
+            handlers::{
+                handle_get_action, handle_get_swap, handle_get_swaps, handle_post_action,
+                handle_post_swap,
+            },
+            socket_addr,
+        },
+    },
+    swap_protocols::{
+        asset::Asset,
+        ledger::{Bitcoin, Ethereum},
+        rfc003::{alice::AliceSpawner, state_store::StateStore, Ledger, SecretSource, Timestamp},
+        MetadataStore, SwapId,
+    },
+};
+use bitcoin_support::BitcoinQuantity;
+use ethereum_support::{Erc20Token, EtherQuantity};
+use http_api_problem::HttpApiProblem;
+use hyper::header;
+use rustic_hal::HalResource;
+use std::{net::SocketAddr, str::FromStr, sync::Arc};
+use warp::{Rejection, Reply};
+
+pub const PROTOCOL_NAME: &str = "rfc003";
+pub fn swap_path(id: SwapId) -> String {
+    format!("/{}/{}/{}", http_api::PATH, PROTOCOL_NAME, id)
+}
+
+#[derive(Debug, Serialize)]
```

Comment:
:wrench: I think some of these structs and enums should go in a separate file or where they are used, like you did with `EmbeddedSwapResource`. Otherwise, I see this file depending on the handlers and the handlers depending on this file.

Reply:
Yes that's fine, will do. I just want to avoid `route_factory.rs` depending on the handlers. We'll see if that happens.

Reply:
e.g. What is the best place for `PROTOCOL_NAME` and `swap_path`? @comit-network/rust-devs

Reply:
`route_factory.rs`?